lib/gym/options.rb

```diff
@@ -101,6 +101,16 @@ def self.plain_options
                                      verify_block: proc do |value|
                                        raise "File not found at path '#{File.expand_path(value)}'".red unless File.exist?(value)
                                      end),
+        FastlaneCore::ConfigItem.new(key: :skip_symbols,
+                                     short_option: "-m",
+                                     env_name: "GYM_SKIP_SYMBOLS",
+                                     description: "Should the ipa file *not* include symbols?",
```

Comment:
The reason for that was, that you can't pass a `false` value from the command line right now.